.github/workflows/build_tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Build tests
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 1 * * 0'
+  pull_request:
+jobs:
+  build-integration-tests:
+    name: Build cli-app
+    runs-on: ubuntu-latest
+    container: mbedos/mbed-os-env:latest
+    steps:
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install -y lavacli curl
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build NRF cli-app
```

Comment:
Not a change request asked, but what would be the impact of using a matrix ? I suppose it would start a `Run integration tests` for each target ?

Reply:
The other job is already a matrix, I'd have to pass another param, this way it's easier, expecially since name of binary is different so I'd need extra code inside this to handle it